js/project.json

```diff
@@ -18,7 +18,8 @@
         "xliffs",
         "resources",
         "package.json",
-        "package-lock.json"
+        "package-lock.json",
+    	"node_modules"
```

Comment:
package-lock.json and package.json are also already in the excluded list from loctool.
so It's okay to remove package.json, package-lock.json, and node_modules.

Reply:
fixed